.reuse/dep5

```diff
@@ -0,0 +1,26 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: YAEMK
+Upstream-Contact: Stefan Kerkmann <karlk90@pm.me>
+Source: https://github.com/KarlK90/yaemk-split-kb
+
+Files: PCB/YAEMK* PCB/assembly/* PCB/*-lib-table PCB/lib/YAEMK.*
+Copyright: Stefan Kerkmann <karlk90@pm.me>
+License: GPL-3.0-only
+
+Files: Bin/*.bin
+Copyright: qmk.fm
+License: GPL-2.0-or-later
+
+Files: PCB/gerbers* docs/yaemk_rev_1_2.html docs/yaemk_*.pdf
+Copyright: Stefan Kerkmann <karlk90@pm.me>
+License: CC-BY-SA-4.0
+
+Files: docs/images/* Case/* PCB/assembly/*
```

Comment:
`PCB/assembly/*` is also listed for GPL-3.0-only in line 6, is this intentional?

Reply:
ahh, good spotting!
will remove it from the GPL-3.0-only section

something else: what is the `PCB/assembly/overrides.json` file.. it is also form you, right?

Reply:
This file is for correcting the parts location files that is generated by KiCAD for assembly by JLCPCB. Mostly rotating the ICs that have a different orientation when uploaded to JLC.